src/jit/codegen.h

```diff
@@ -464,7 +464,8 @@ protected :
                                     INDEBUG_LDISASM_COMMA(CORINFO_SIG_INFO* sigInfo)
                                     void*                 addr
                                     X86_ARG(ssize_t       argSize),
-                                    emitAttr              retSize,
+                                    emitAttr              retSize
+                                    FEATURE_UNIX_AMD64_STRUCT_PASSING_ONLY_ARG(emitAttr retSize1),
```

Comment:
I would much prefer to see this abstracted to take an argument that includes the retSize for the maximum number of return registers for a give target architecture.